### DISCUSSION TITLE
how do I access other topics of discussions.

### DISCUSSION THREAD

**Discussion by nibin-matteo-git** [Homebrew/discussions#1310] [Everyday usage]
Just got introduced to github discussions.
Is this just like stack overflow.
If so is there anyway to discuss about other topics like python or other cs related stuff cuz I can't find any other descussions other than the one with homebrew.
<img width="667" alt="Screenshot 2021-04-19 at 1 07 04 PM" src="https://user-images.githubusercontent.com/80332406/115198828-24918f80-a110-11eb-9854-ef9b18320bf3.png">

Is there anyway to access these discussions cuz i go to 'your discussions' every time and then navigate.

**Comment by SMillerDev** [comment]
> Is this just like stack overflow.

It's more like discourse.brew.sh.

> If so is there anyway to discuss about other topics like python or other cs related stuff cuz I can't find any other descussions other than the one with homebrew.

The homebrew discussions page is really only for discussions about homebrew.

**Reply by nibin-matteo-git** [reply]
so is there any other page for python
if so how do I get there

**Reply by SMillerDev** [reply]
https://bfy.tw/QlPC

**Reply by nibin-matteo-git** [reply]
what I mean is,
how do i get to the page.
For example when i google python discussions github I the link which takes me to the page.
But how do we access it from within github.
And yes I did google it before I came here 😂

**Reply by carlocab** [reply]
You can use the GitHub search bar on the top left of the page. You can jump to it by pressing `/`.
